requirements/philu/base.txt

```diff
@@ -10,4 +10,5 @@ imgkit==1.0.1
 pdfkit==0.6.1
 -e git+https://github.com/philanthropy-u/xblock-poll#egg=xblock-poll
 -e git+https://github.com/philanthropy-u/edx-notifications.git@9c602923d0e8a7b512a52bc4d0df5c7043286386#egg=edx-notifications
+-e git+https://github.com/raccoongang/edx_xblock_scorm.git#egg=edx_xblock_scorm
```

Comment:
Also I hope we have tested its compatibility with ironwood.master. If not then let's please make sure it's not upgrading/downgrading any existing package. And one way to test it is to compare the pip_freeze logs before adding this package and after adding this package.

Reply:
Also, is there any specific reason to install this package in editable mode?

Reply:
@ahmed-faraz46 it is already in philu/base.txt isn't it?? yes, I have check it is not updating any requirement as per my observation. it would be great if you can check in your env as well to counter verify. And no specific reason following the practice as it is already followed in the project.

Reply:
@qasimgulzar My bad, 1) yes it's already in philu/base.txt. 2) Sure, I'll reverify and let you know. 3) It's not a practice `-e` is there as we have customized all those packages. Please notice all of them are forked in philanthropyu and are customized. But we are directly using this package without customizing anything , try to use pypi link (specify version) instead of installing directly from github, add a TODO to replicate it in testing.py after LP-2136 is merged.

Reply:
Sounds good! and reverified it's not updating any existing dependency 🎉